plantseg/utils.py

```diff
@@ -276,3 +277,14 @@ def clean_models():
 
         else:
             print("Invalid input, please type 'y' or 'n'.")
+
+
+def check_version():
+    plantseg_url = ' https://api.github.com/repos/hci-unihd/plant-seg/releases/latest'
+    response = requests.get(plantseg_url).json()
+    latest_version = response['tag_name']
+    latest_version_numeric = int(latest_version.replace('.', ''))
+    plantseg_version_numeric = int(current_version.replace('.', ''))
+    if latest_version_numeric > plantseg_version_numeric:
```

Comment:
this might not always work though :)
`v2.0 > v1.5.1`, but this code will convert it to `20 > 151` which is not correct. But assuming we always have `major.minor.patch` from the `bumpversion` this should be fine